src/main/GameOfLife.java

```diff
@@ -52,7 +53,7 @@ private GameBoard generate(GameBoard gameBoard) {
 
     private Cell processStrategiesForCell(int rowNumber, int colNumber, GameBoard gameBoard) {
         Cell processedCell = new Cell();
```

Comment:
It's a little weird that you create the cell out of the strategy and it modifies the cell. I think that the strategy could return the new cell, but it is my point of view. Is not bad, but is a little weird